schemas/rally-zero-one/measurements/measurements.1.schema.json

```diff
@@ -0,0 +1,104 @@
+          "type": "string"
+        },
+        "eventType": {
+          "description": "the type of event recorded. Either \"attention\" or \"audio\". An attention event is an instance where the user was actively using the browser in an active tab in an active window. An audio event tells us when an active browser tab has audio playing. We use this as a proxy for a user passively consuming audio and video. These two event types have near identical payloads, with a few exceptions noted below.",
+          "enum": [
+            "attention",
+            "audio"
+          ],
+          "type": "string"
+        },
+        "maxPixelScrollDepth": {
+          "description": "The largest scroll pixel depth reached on the page",
+          "type": "integer"
+        },
+        "maxRelativeScrollDepth": {
+          "description": "The largest depth reach on the page, as a proportion of the total page height",
+          "type": "number"
+        },
+        "ogType": {
+          "description": "the og:type meta tag contents (e.g. <meta type=\"og:type\" contents=\"article\" />)",
+          "type": "string"
+        },
+        "origin": {
+          "description": "the origin of the URL associated with the page visit. Calculated by applying new URL(url).origin. See https://developer.mozilla.org/en-US/docs/Web/API/URL/origin",
+          "type": "string"
+        },
+        "pageId": {
+          "description": "a unique ID associated with a page visit. Each page ID is 128-bit value, randomly generated with the Web Crypto API and stored as a hexadecimal `String`. While this representation is less efficient than a `Uint8Array` or similar, it is more convenient for development and debugging.",
+          "type": "string"
+        },
+        "pageVisitStartTime": {
+          "description": "unix timestamp (in ms) of the page visit start",
+          "type": "integer"
+        },
+        "pageVisitStopTime": {
+          "description": "unix timestamp (in ms) of the page visit end. NOTE: this field will not necessarily represent the page visit stop time, just the largest time value at the time of the event creation. For a given page id, look for the largest value of pageVisitStopTime to get more accurate information.",
+          "type": "integer"
+        },
+        "referrerOrigin": {
+          "description": "the origin of the referrer URL for the page loading in the tab, or `\"\"` if there is no referrer.",
+          "type": "string"
+        },
+        "scrollHeight": {
+          "description": "the total scroll height of the page, taken from document.documentElement.scrollHeight at the same interval as the other scroll fields.",
+          "type": "number"
+        },
+        "title": {
+          "description": "the contents of the title element in the head of the page",
+          "type": "string"
+        }
+      },
+      "required": [
+        "eventType",
+        "pageId",
+        "url",
+        "referrer",
```

Comment:
hmm, I think this is wrong. This should be `referrerOrigin` and `url` should be `origin`.

Reply:
I believe this would be something that would have been caught with an example document :) One would be much appreciated, otherwise I'll throw something together before I merge this in.

Reply:
Yes, I didn't realize this was a requirement but agreed that this would be helpful. I'll generate two different ones to test against. Thank you for the suggestion.